src/components/evaluation/EvaluationActions.vue

```diff
@@ -187,19 +187,7 @@ export default {
     }
   },
   methods: {
-    afterApply(response) {
-      this.refreshAll().then(() => {
-        const selectedRowCount = this.applyingAction.key === 'duplicate' ? ((response.length || 0) / 2) : (response.length || 0)
-        const target = `${selectedRowCount} ${selectedRowCount === 1 ? 'row' : 'rows'}`
-        this.alertScreenReader(`${this.applyingAction.completedText} ${target}`)
-        this.$putFocusNextTick('select-all-evals-checkbox')
-        this.reset()
-      }).finally(() => {
-        this.isApplying = false
-        this.setDisableControls(false)
-      })
-    },
```

Comment:
`afterApply()` (above) and `handleError()` have been incorporated into the single code block of `update()` (below).